### ISSUE TITLE
Wrong padding of "label"

### ISSUE THREAD

**Issue by shaibagon** [jfzhang95/pytorch-deeplab-xception#50]
When training, the augmentation `RandomScaleCrop` may downscale the image and the target label image. It then pads the image and the label with [`self.fill`][1] which is ZERO.  
This is in contrast to the "ignore value" of the loss [that is set to 255][2].  
This way the loss treats the padded region as valid "class 0" pixels and compute loss for it.

`self.fill` of the augmentation functions should be equal to `self.ignore_index` of the loss function.

 [1]: https://github.com/jfzhang95/pytorch-deeplab-xception/blob/master/dataloaders/custom_transforms.py#L113
 [2]: https://github.com/jfzhang95/pytorch-deeplab-xception/blob/master/utils/loss.py#L5
